src/fu-usb-device.c

```diff
@@ -388,6 +388,55 @@ fu_usb_device_set_dev (FuUsbDevice *device, GUsbDevice *usb_device)
 				   g_usb_device_get_platform_id (usb_device));
 }
 
+/**
+ * fu_usb_device_find_udev_device:
+ * @device: A #FuUsbDevice
+ * @usb_device: A #GUsbDevice, or %NULL
+ *
+ * Gets the matching #GUdevDevice for the #GUsbDevice.
+ *
+ * Returns: a #GUdevDevice, or NULL if unset or invalid
+ *
+ * Since: 1.3.1
```

Comment:
1.3.2

Reply:
I will change this in the next commit

Reply:
If you can push this in the next couple of days I can include this plugin in the 1.3.2 release.

Reply:
@hughsie I have been busy with our next generation of laptops but I will try to push these changes today